ganghee/MyStudy2/app/src/main/java/com/example/mystudy/adapter/UpbitHolder.kt

```diff
@@ -1,25 +1,13 @@
 package com.example.mystudy.adapter
 
-import android.view.View
-import android.widget.TextView
-import androidx.core.content.ContextCompat
 import androidx.recyclerview.widget.RecyclerView
 import com.example.mystudy.data.FormatTickers
-import kotlinx.android.synthetic.main.rv_item_list.view.*
+import com.example.mystudy.databinding.RvItemListBinding
 
-class UpbitHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-    private val coinName: TextView = itemView.tv_coin_name
-    private val tradePrice: TextView = itemView.tv_trade_price
-    private val signedChangeRate: TextView = itemView.tv_signed_change_rate
-    private val accTradePrice24th: TextView = itemView.tv_acc_trade_price_24h
+class UpbitHolder(private val binding: RvItemListBinding) : RecyclerView.ViewHolder(binding.root) {
```

Comment:
클래스 관련 convention을 따르는 것을 권장합니다.
https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting

Reply:
수정하였습니다.
[UpbitHolder](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/195/commits/368adf0d1228d7476ab60394d047867d1be367ef)

Reply:
다음과 같이 사용하는 것이 좋을 것 같습니다.
```suggestion
class UpbitHolder(
    private val binding: RvItemListBinding
) : RecyclerView.ViewHolder(binding.root) {
```

Reply:
class에 관한 convention을 적용하여 바꿔보았습니다.
[UpbitHolder](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/195/commits/e8d145432491cdaae3d20f5a17f222d15d928eed)